packages/emitter-framework/src/python/components/class-variable.tsx

```diff
@@ -0,0 +1,16 @@
+import { code } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js";
+
+export interface ClassVariableProps {
+    name: string;
+    // TODO: Replace this with a component so we can handle imports?
+    type: string;
+}
+
+export function ClassVariable(props: ClassVariableProps) {
+  const name = usePythonNamePolicy().getName(props.name, "classMember");
```

Comment:
Wondering if thats always how you call if having a `usePythonName(props.name, "classMember")` would not be nicer

Reply:
I like this idea!